server.js

```diff
@@ -0,0 +1,49 @@
+// code adapted from https://cloud.google.com/translate/docs/basic/translating-text#translate_translate_text-nodejs
+// Imports the Google Cloud client library
+import ws from 'ws';
+import express from 'express';
+import http from 'http';
+import fs from 'fs';
+import Translate from '@google-cloud/translate';
+const translator = new Translate.v2.Translate();
+
+// setup express server
+const app = express();
+app.use(express.static('client'));
+const server = http.createServer(app);
+
+// create api key file for GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
The comment here is required, but you're not answering the question that the next developer (or even future you) will be asking.

Your comment describes the if else statement that follows, which explains itself anyway. Your comment here should be explaining *why* this is necessary – i.e, google module needs to read the credentials from a file instead of directly from the environment variable, etc.